docs/common/routes.ts

```diff
@@ -68,10 +68,13 @@ export const getPageSection = (path: string) => {
 };
 
 export const getCanonicalUrl = (path: string) => {
+  console.log(path);
```

Comment:
```suggestion
```

Let's remove the console log, and also make sure that we have the correct setting of `traillingSlash` set for Next: 
* https://nextjs.org/docs/pages/api-reference/next-config-js/trailingSlash

Reply:
oops! Good catch! Thank you!

We do have trailingSlash setting enabled in `next.config.js` in our docs repo:

![CleanShot 2024-01-02 at 15 53 28@2x](https://github.com/expo/expo/assets/10234615/1dce89f7-e53a-46ac-aced-9953cc885c46)